test/example/type-builder.cpp

```diff
@@ -1,233 +0,0 @@
-#include <cassert>
-#include <iostream>
-
-#include "wasm-type-printing.h"
-#include "wasm-type.h"
-
-using namespace wasm;
-
-// Check that the builder works when there are duplicate definitions
-void test_canonicalization() {
```

Comment:
Is this test no longer needed? Not all tests in this file test constructing basic types AFAICT.

Reply:
Everything else here is already well tested in test/gtest/type-builder.cpp.